src/app.js

```diff
@@ -267,6 +268,10 @@ import("./translations/en").then((en) => lang.init(en.default)).then(async () =>
 	// see https://github.com/MithrilJS/mithril.js/issues/2659
 	m.route.prefix = neverNull(state.prefix).replace(/(?:%[a-f89][a-f0-9])+/gim, decodeURIComponent)
 
+	if (isApp() || isDesktop()) {
```

Comment:
I would prefer if we:
 - described here why it should be here
 - described in the commit why it wasn't working